charts/emissary-ingress/templates/ambassador-agent.yaml

```diff
@@ -65,6 +65,9 @@ rules:
 - apiGroups: [""]
   resources: [ "configmaps" ]
   verbs: [ "get", "list", "watch" ]
+- apiGroups: [""]
```

Comment:
- Yes, the roles are either global or by namespace. When it's by namespace, you can't do anything out of it.
- The agent only creates secrets. The API Keys & Filters are created by the end user using GitOps or manual creation.
- We were planning to only create filters in the ambassador namespace, but I don't know if we should support filters created in a different namespace by the end user. In this case, you're right, this permission may not be enough.

Reply:
In general, we own the Ambassador namespace so its best if users place stuff in their own namespace. I would think if we were creating the filters on their behalf via the cloud then maybe it makes sense to put them there or some new namespace specific to this use case. 

But in this case if we are creating the `secrets` in our namespace then based on the recommendation of removing the `namespace` from the `APIKeyFilter` will not work if a user creates a Filter in another namespace, correct? Because, we require the secret to be in the same namespace?

Reply:
understood